rust/azure_iot_operations_connector/Cargo.toml

```diff
@@ -12,9 +12,9 @@ readme = "README.md"
 publish = true
```

Comment:
(can't comment on that line, but) should version here be updated to 
`version = "0.1.0-alpha.1"`
similar to the other crates?

Reply:
Yes, good catch. I thought I did that but evidently not.